crowbar_framework/app/models/api/upgrade.rb

```diff
@@ -24,6 +24,10 @@ def status
       end
 
       def checks
+        upgrade_status = ::Crowbar::UpgradeStatus.new
+        # the check for current_step means to allow running the step at any point in time
+        upgrade_status.start_step if upgrade_status.current_step == "upgrade_prechecks"
```

Comment:
@MaximilianMeister there's one more check for "upgrade_prechecks" (string, not symbol)